wireapp/wire-android-reloaded
# fix: mute/unmute visuals displayed with delay for self user

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

mute/unmute visuals displayed with delay for self user.

### Causes (Optional)

We were using values returned by AVS

### Solutions

use local values for current user and AVS values for other participants 

Needs releases with:


### Testing

#### How to Test

Make a call and try to mute or unmute yourself
visuals state should change immediately without any delay

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
